install/install_openadapt.sh

```diff
@@ -31,6 +139,6 @@ pip install -r requirements.txt
 #   [notice] To update, run: pip install --upgrade pip
 pip install -e .
 
-RunAndCheck "alembic upgrade head"
-RunAndCheck "python3.10 -m spacy download en_core_web_trf"
-RunAndCheck "pytest"
\ No newline at end of file
+RunAndCheck "alembic upgrade head" "Update database"
+RunAndCheck "python3.10 -m spacy download en_core_web_trf" "Download spacy model"
```

Comment:
We can safely remove the installation of trf model for now as it is already included in` poerty install.`

Reply:
Addressed in [c90084b](https://github.com/MLDSAI/OpenAdapt/pull/244/commits/c90084b140e4b19c52e862da817cc22f3c3acc40)